### DISCUSSION TITLE
¿Cómo cambiar el concepto de compra en Mercado pago para personalizar con nombre de empresa?

### DISCUSSION THREAD

**Discussion by jonathan-easycodigo** [mercadopago/sdk-js#84] [General]
Buen dia,

Quisiera saber como cambiar el concepto que llega en las compras a mis clientes, pues dice Compra en Mercado Pago y creo que se debería poder personalizar a Compra en X Negocio.

¿Es posible hacerlo? ¿Como puedo hacerlo?

Muchas gracias!

**Comment by lucmantovani** [comment]
Hola, @jonathan-easycodigo!

No estoy seguro de que entendí lo que necesitas... Pero de cualquier manera, para dudas de integración el canal recomendado es el [soporte](https://www.mercadopago.com.mx/developers/es/support/center/tickets).

Saludos!
